certbot fails on darwitn with
```
ERROR: test_no_persisted_empty (simp_le.ExternalIOPluginTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/nix/store/5myi8sjqdhii0wsrcrjaw1pkzc14hmnj-simp_le-client-0.8.0/lib/python2.7/site-packages/simp_le.py", line 886, in test_no_persisted_empty
    self.assertEqual(IOPlugin.EMPTY_DATA, self.plugin.load())
  File "/nix/store/5myi8sjqdhii0wsrcrjaw1pkzc14hmnj-simp_le-client-0.8.0/lib/python2.7/site-packages/simp_le.py", line 654, in load
    pems = list(self.split_pems(self.get_output_or_fail('load')))
  File "/nix/store/5myi8sjqdhii0wsrcrjaw1pkzc14hmnj-simp_le-client-0.8.0/lib/python2.7/site-packages/simp_le.py", line 629, in get_output_or_fail
    raise Error('Failed to execute external script: {0}'.format(error))
Error: Failed to execute external script: [Errno 8] Exec format error
```
not sure how to fix it. should i disable this test for darwin? everything else looks fine

--- a/pkgs/tools/admin/simp_le/default.nix
+++ b/pkgs/tools/admin/simp_le/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub, fetchpatch, pythonPackages }:
+{ stdenv, fetchFromGitHub, fetchpatch, pythonPackages, bash }:
  
 pythonPackages.buildPythonApplication rec {
   pname = "simp_le-client";
@@ -8,6 +8,11 @@ pythonPackages.buildPythonApplication rec {
     inherit pname version;
     sha256 = "0nv9mm99rm8i9flgfgwvmajbsxb5rm162nfxlq3wk66bbbyr6y1i";
   };
+
+  postPatch = ''
+    substituteInPlace simp_le.py \
+      --replace "/bin/sh" "${bash}/bin/sh"
+  '';
 
   checkPhase = ''
     $out/bin/simp_le --test

